index.html

```diff
@@ -44,23 +46,23 @@ <h2 id="name">I'm David Verde! Glad to see you! </h2>
         <ul>
           <li>
             <a href="https://twitter.com/UnyieldingOne">
-              <img src="/Image/twitter.svg" alt=""></a>
+              <img src="/Portfolio-setup-and-mobile-version-skeleton/Image/twitter.svg" alt="twitter"></a>
           </li>
           <li>
             <a href="https://www.linkedin.com/in/david-verde-3349b114b/e">
-              <img id="Vector" src="/Image/Vector.svg" alt=""></a>
+              <img id="vector" src="/Portfolio-setup-and-mobile-version-skeleton/Image/Vector.svg" alt="linkedin"></a>
           </li>
           <li>
             <a href="https://medium.com/@david.verde.alvarez">
-              <img src="/Image/medium.svg" alt=""> </a>
+              <img src="/Portfolio-setup-and-mobile-version-skeleton/Image/medium.svg" alt="medium"> </a>
           </li>
           <li>
             <a href="https://github.com/Unyielding1">
-              <img src="/Image/github.svg" alt=""> </a>
+              <img src="/Portfolio-setup-and-mobile-version-skeleton/Image/github.svg" alt="github"> </a>
           </li>
           <li>
             <a href="https://angel.co/profile/edit/overview">
-              <img src="/Image/angellist.svg" alt=""> </a>
+              <img src="/Portfolio-setup-and-mobile-version-skeleton/Image/angellist.svg" alt="angellist"> </a>
```

Comment:
- [ ] Similar to the previous comment. Kindly use relative path instead of an absolute path
- [ ] Kindly use use flex to list the icons instead of inline